draft-ietf-dnsop-structured-dns-error.md

```diff
@@ -308,11 +308,19 @@ and diagnosing the cause of the DNS filtering.
 
 ## Client Generating Request {#client-request}
 
-When generating a DNS query, the client includes the Extended DNS
-Error option Section 2 of {{!RFC8914}} in the OPT pseudo-RR {{!RFC6891}} to
-elicit the Extended DNS Error option in the DNS response. It SHOULD use an
-option-length of 0 (that is, omitting INFO-CODE and EXTRA-TEXT from
-{{!RFC8914}}).
+To prevent the EDE Option attacks to an EDE OPT-unaware DNS responder
```

Comment:
I don't parse this one.

Reply:
it appears you added () in a later commit to clarify the confusion in this earlier commit, so I am marking this resolved.